core/src/main/scala/kafka/metrics/KafkaMetricsReporter.scala

```diff
@@ -52,12 +52,11 @@ trait KafkaMetricsReporter {
 }
 
 object KafkaMetricsReporter {
-  val ReporterStarted: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
Why does it never get set to `true`? If you configure `KafkaMetricReporters`, it should be set to `true` right?